app/helpers/forever_style_guide/application_helper.rb

```diff
@@ -265,40 +261,24 @@ def new_digital_art_url
       www_url('/digital_art_library?facet_name=new')
     end
 
-    def free_digital_art_url
-      www_url('/digital_art_library?facet_name=free')
-    end
-
-    def all_digital_art_url
-      www_url('/digital_art_library')
-    end
-
-    def print_url
-      www_url('/print')
-    end
-
-    def photo_books_url
-      www_url('/photobooks')
-    end
-
-    def cards_url
-      www_url('/cards')
+    def all_photo_books_url
+      www_url('/prints_library')
     end
 
-    def calendars_url
-      www_url('/calendars')
+    def featured_photo_books_url
+      www_url('/prints_library?facet_name=print-featured')
     end
 
-    def wall_art_url
-      www_url('/wall_art')
+    def free_digital_art_url
+      www_url('/digital_art_library?facet_name=free')
     end
 
-    def photo_gifts_url
-      www_url('/photo_gifts')
+    def predesigned_layouts_digital_art_url
+      www_url('/digital_art_library?facet_name=page-layouts')
```

Comment:
I think this should be more like this
www.forever.com/digital_art_library?facet_name=pre-designed%20layouts

Reply:
I was following the convention of using the name as in other references here but checked in with Brad and it is preferred to use the slug so this has been updated

Reply:
the name would work, but there is no facet named page layouts
https://www.forever.com/digital_art_library?facet_name=page-layouts